src/elements/content-sidebar/DocgenSidebar.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+/**
+ * @flow
+ * @file Doc Gen sidebar component
+ * @author Box
+ */
+
+import flow from 'lodash/flow';
+import * as React from 'react';
+import type { ErrorContextProps } from '../../common/types/api';
+import type { WithLoggerProps } from '../../common/types/logging';
+import { ORIGIN_METADATA_SIDEBAR, SIDEBAR_VIEW_METADATA } from '../../constants';
+import { mark } from '../../utils/performance';
+import { withAPIContext } from '../common/api-context';
+import { withErrorBoundary } from '../common/error-boundary';
+import { withLogger } from '../common/logger';
+import { EVENT_JS_READY } from '../common/logger/constants';
+import SidebarContent from './SidebarContent';
+
+type ExternalProps = {
+    enabled: boolean,
+    getDocGenTags: Function,
+};
+
+type DocGenTag = {
+    json_paths: Array<string>,
+    tag_content: string,
+    tag_type: string,
+};
+
+type Props = {
+    isLoading: boolean,
+} & ExternalProps &
+    ErrorContextProps &
+    WithLoggerProps;
+
+type State = { tags: Array<DocGenTag> };
+
+const MARK_NAME_JS_READY = `${ORIGIN_METADATA_SIDEBAR}_${EVENT_JS_READY}`;
+
+mark(MARK_NAME_JS_READY);
+
+// TO DO: implement actual sidebar content in separate tickets
+class DocGenSidebar extends React.PureComponent<Props, State> {
+    state = {
+        tags: [],
+    };
+
+    componentDidMount() {
+        if (this.props.getDocGenTags) {
+            this.props.getDocGenTags().then(response => {
+                this.setState({ tags: response?.payload?.data });
+            });
+        }
+    }
+
+    render() {
+        const { isLoading } = this.props;
+        const { tags } = this.state;
+
+        return (
+            <SidebarContent sidebarView={SIDEBAR_VIEW_METADATA} title="Box DocGen">
+                <input placeholder="Search" />
+                {isLoading && <div>Loading</div>}
```

Comment:
translation for this one as well?

Reply:
the purpose of this PR is to display sidebar tab by default and add a wrapper page for the Docgen Sidebar - all of the content inside DocgenSidebar component is just dummy content that will be reworked separately.

I'm trying to split the epic I'm working on into small units of work that are easy to review instead of having entire epic in 1 PR

Reply:
great!